src/osd/PGLog.h

```diff
@@ -1115,10 +1115,10 @@ struct PGLog : DoutPrefixProvider {
     const pg_info_t &info,
     IndexedLog &log,
     missing_type &missing, ostringstream &oss,
-    bool *clear_divergent_priors = NULL,
-    const DoutPrefixProvider *dpp = NULL,
-    set<string> *log_keys_debug = 0,
-    bool debug_verify_stored_missing = false
+    bool *clear_divergent_priors,
+    const DoutPrefixProvider *dpp,
+    set<string> *log_keys_debug,
+    bool debug_verify_stored_missing
     ) {
```

Comment:
The call site at ceph_objectstore_tool.cc:331 (there mat be others) relies on the defaults being set.